lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -953,6 +953,9 @@ public function get_item_schema() {
 					'description'     => __( 'The content for the object.' ),
 					'type'            => 'object',
 					'context'         => array( 'view', 'edit', 'embed' ),
+					'arg_options' => array(
+						'sanitize_callback' => null,
+					),
```

Comment:
I'd like to see a `// polymorphic value: sanitization implemented in controller` or some such on these, so that the rationale for this otherwise inscrutable change doesn't get lost